renderers/simple.go

```diff
@@ -88,3 +88,10 @@ func (r SimpleRenderer) ScopeHasStarted(scopes []string) bool {
 	_, result := r.startTimes[timeKey]
 	return result
 }
+
+func quotedIfNeeded(s string) string {
```

Comment:
A few tests would be nice, e.g. is `Task 'a'` a valid target for quoting or not?

Reply:
See d34d80eee78f22dfe97b00dab192ab3515e26736